.vscode/c_cpp_properties.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Can we remove this by adding in gitignore?

Reply:
we need this so vscode doesn't throw error squiggles

Reply:
if you are using another editor feel free to ignore, I don't think it affects stuff